docs/src/time_integration.md

```diff
@@ -14,3 +14,13 @@ Interesting classes of time integration schemes are
   of stages, e.g. to allow for interpolation (dense output), root-finding for continuous callbacks,
   and error-based time step control. In general, you often should not need to worry about this if you
   use Trixi.
+
+Some common options for `solve` are the following. Further documentation can be found
+in the [SciML docs](https://diffeq.sciml.ai/v6.8/basics/common_solver_opts/).
```

Comment:
Maybe to make it clearer:
```suggestion
Some common options for `OrdinaryDiffEq.solve` are the following. Further documentation can be found
in the [SciML docs](https://diffeq.sciml.ai/v6.8/basics/common_solver_opts/):
```

Reply:
I've made a similar change in a72a1e2. I don't like to write `OrdinaryDiffEq.solve`, since `solve` comes actually from https://github.com/SciML/CommonSolve.jl.

Reply:
👍